app/components/section-title/template.hbs

```diff
@@ -1,15 +1,19 @@
-{{#if layout.settings.useLogo}}
-    <div
-        class="coll-hero-logo"
-        style="{{logoStyle}}"
-        aria-role="img"
-        aria-label="{{branding.logo.label}}"
-    ></div>
-{{else if layout.title}}
-    <h1 style="{{titleColor}}">{{layout.title}}</h1>
-{{else}}
-    <h1>{{model.title}}</h1>
-{{/if}}
+<div class="title-wrapper">
+
+    {{#if layout.logo}}
+            <img class="title-logo" src='{{layout.logo}}' alt="logo"/>
```

Comment:
It might make more sense to put the logo inside of the h1 as the logo is part of the heading. This avoids the title-wrapper also.

Reply:
I think there are also a couple examples of using spans with the image set as the background image around the repo - this makes it a little easier to size and keep aspect ratio